yb-voyager/cmd/importData.go

```diff
@@ -933,12 +933,19 @@ func executeSqlFile(file string) {
 
 	var errOccured = 0
 	sqlStrArray := createSqlStrArray(file, "")
-	for _, sqlStr := range sqlStrArray {
+	retryCount := INDEX_RETRY_COUNT
```

Comment:
Let's not call it as a "index retry count". We're going to retry any DDL statement if it fails with a "retriable" error. For now, the only retriable error that we know of is "Query error: schema version mismatch".

We can call it something like `DDL_STATEMENT_MAX_RETRY_COUNT`.

Reply:
done